python/vineyard/cli.py

```diff
@@ -67,47 +86,62 @@ def connect_vineyard(args):
     else:
         exit_with_help()
 
-    global __vineyard_client
-    __vineyard_client = client
```

Comment:
Are these two lines have some desired side effects? @sighingnow

Reply:
Its ok to delete those lines.